### PR TITLE
IM-207: Implemented breadcrumbs

### PR DESCRIPTION
**Pull request by hinyunlee**
Some style improvements over every page. Implemented new breadcrumbs API
in navigation. Now has global back function. Back button removed and
it's function is replaced by the history links on top.

### CODE REVIEW

**Comment by hinyunlee** [comment]
Includes IM-251.

**Comment by mrstarware** [comment]
View Jobs doesn't display any jobs.

**Comment by hinyunlee** [comment]
View jobs doesn't display any jobs because i combined IM-174 to this ticket. It shows jobs for the current user logged on. That means you need to make a new cart. However, it does not show the correct cart because the backend does not handle that correctly yet.

File: public/javascripts/login.js [hunk 1]
```diff
@@ -1,43 +1,65 @@
...
+                                    console.log(err);
+                                }
+
+                                if (user.PermsID == 604) {
+                                    $("#AdminBar").removeClass("hidden");
+                                    $("#AdminLogs").removeClass("hidden");
+                                    $("#AddUsers").removeClass("hidden");
+                                    $("#DeleteUsers").removeClass("hidden");
+                                    $("#ViewUsers").removeClass("hidden");
+                                }
```
Comment (mrstarware) [inline public/javascripts/login.js:50]: I think this needs to be updated. My ability to see Manage Users disappeared.

Other than this fix. LGTM
Good job dude!

**Pull request merged**